src/icons/AnonFace.tsx

```diff
@@ -0,0 +1,76 @@
+import classnames, {
+  dropShadow,
+  stroke,
+  strokeWidth,
+} from 'classnames/tailwind'
+
+const svgStyle = classnames(dropShadow('drop-shadow-blue-light'))
```

Comment:
Get rid of `classnames`, use can use `dropShadow(..)` without `classnames` wrapper, you need to wrap when you have multiple properties